tests/services/events/test_api.py

```diff
@@ -5,7 +5,7 @@
 
 import pytest
 import tornado
-from jupyter_telemetry.eventlog import _skip_message
+from jupyter_events.logger import _skip_message
```

Comment:
Should `_skip_message` be a "public" function in `jupyter_events`?

Reply:
This was removed in the last commit. Thanks!